code/modules/client/preference/preferences.dm

```diff
@@ -2149,7 +2149,6 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	if(disabilities & DISABILITY_FLAG_FAT && character.species.flags & CAN_BE_FAT)
 		character.dna.SetSEState(FATBLOCK,1,1)
-		character.mutations += FAT
```

Comment:
Resolved by removing the redundant character mutation and disabilities setting, you were right about the SetSEState doing it already. Didn't need to add a genemutcheck since it's already done below where I marked with a comment